src/dune_rules/odoc.ml

```diff
@@ -218,6 +248,21 @@ let compile_mld sctx (m : Mld.t) ~includes ~doc_dir ~pkg =
        ]);
   odoc_file
 
+let compile_mld_to_odocl sctx (m : Mld.t) ~includes ~doc_dir ~pkg =
```

Comment:
Same with this

Reply:
Instance of https://github.com/ocaml/dune/pull/3927#discussion_r523978857